Flat/src/PandaAnalyzer.cc

```diff
@@ -78,9 +78,9 @@ PandaAnalyzer::PandaAnalyzer(Analysis* a, int debug_/*=0*/) :
   TH1D* hDTotalMCWeight = static_cast<TH1D*>(static_cast<TH1D*>(fIn->Get("hSumW"))->Clone("hDTotalMCWeight"));
   hDTotalMCWeight->SetDirectory(0);
   TH1D* hDNPUWeight = nullptr; {
-    TH1D* hbase = static_cast<TH1D*>(fIn->Get("hNPVReco"));
+    TH1D* hbase = static_cast<TH1D*>(fIn->Get("hNPVTrue"));
```

Comment:
We want something like this no?
    TH1D* hbase = static_cast<TH1D*>(fIn->Get("hNPVTrue"));
    if (hbase == nullptr)
      hbase = static_cast<TH1D*>(fIn->Get("hNPVReco"));

In this way, it will plot hNPVTrue for MC, no?

Reply:
Yes sorry, I'm getting confused by the git diff. 

Something is wrong - the diff shows that the current master has Reco first, but that's clearly not the case [1]. I'll sort it out interactively.

[1] https://github.com/sidnarayanan/PandaAnalysis/blob/5ca629dff8f96172d90dcac0eb3f39e0b69d0aea/Flat/src/PandaAnalyzer.cc#L82

Reply:
Right, this is what I'm telling you. The master version is wrong in my opinion